features/step_definitions/plans/account_plans_steps.rb

```diff
@@ -101,9 +101,9 @@
 
 When "an admin is looking for an account plan" do
   AccountPlan.destroy_all
-  FactoryBot.create(:account_plan, provider: @provider, name: 'This is number One')
-  FactoryBot.create(:account_plan, provider: @provider, name: 'Now the second one', state: 'published')
-  FactoryBot.create(:account_plan, provider: @provider, name: 'Finally the Last')
+  FactoryBot.create(:account_plan, provider: @provider, name: 'First')
+  FactoryBot.create(:account_plan, provider: @provider, name: 'Second', state: 'published')
```

Comment:
Why renaming all the new items?

Reply:
I decided to rename those items because they were causing conflicts with the search that already existed in master.

Reply:
The idea behind those quirky names is that they allow searching in pairs. For instance `one` and `the` will return 2 different combinations.
What was the conflict with master?